runtime/browser/runtime_download_manager_delegate.cc

```diff
@@ -164,7 +164,7 @@ void RuntimeDownloadManagerDelegate::ChooseDownloadPath(
   if (GetSaveFileName(&save_as))
     result = base::FilePath(std::wstring(save_as.lpstrFile));
 #else
-  NOTIMPLEMENTED();
+  result = suggested_path;
```

Comment:
It will change the Crosswalk behavior in many other platforms beside the Android.